src/Processors/ExecutionStartingProcessor.cs

```diff
@@ -44,7 +45,8 @@ public override Message Process(Message request)
             if (debuggingEnv != null && debuggingEnv.ToLower().Equals("true"))
             {
                 // if the runner is launched in DEBUG mode, let the debugger attach.
-                System.Console.WriteLine("Runner Ready for Debugging at Process ID " + System.Diagnostics.Process.GetCurrentProcess().Id);
+                Console.WriteLine("Runner Ready for Debugging at Process ID " +
+                                  System.Diagnostics.Process.GetCurrentProcess().Id);
```

Comment:
can be `Process.GetCurrentProcess` since `System.Diagnostics` is already in `using`.

Reply:
That creates name collision with the method name (Process).